src/core/Akka.Streams/Dsl/FlowOperations.cs

```diff
@@ -2438,5 +2438,42 @@ public static class FlowOperations
 
             return FlowWithContext.From(flowWithTuples);
         }
+        
+        /// <summary>
+        /// Repeats the previous element from upstream until it's replaced by a new value.
+        /// </summary>
+        /// <param name="flow">The previous Flow stage in this stream.</param>
+        /// <typeparam name="TIn">The input type.</typeparam>
+        /// <typeparam name="TOut">The output type.</typeparam>
+        /// <typeparam name="TMat">The materialization type.</typeparam>
+        /// <remarks>
+        /// This is designed to allow fan-in stages where output from one of the sources is intermittent / infrequent
+        /// and users just want the previous value to be reused.
+        /// </remarks>
+        public static Flow<TIn, TOut, TMat> RepeatPrevious<TIn, TOut, TMat>(this Flow<TIn, TOut, TMat> flow)
+        {
+            return flow.Via(new ReuseLatest<TOut>());
+        }
+
+        /// <summary>
+        /// Repeats the previous element from upstream until it's replaced by a new value.
+        /// </summary>
+        /// <param name="flow">The previous Flow stage in this stream.</param>
+        /// <param name="onItemChanged">A <see cref="Action{TOut, TOut}"/> function that allows the stage to perform clean-up operations when the previously repeated
+        /// value is being replaced.
+        /// 
+        /// This is used for things like calling <see cref="IDisposable.Dispose"/> on the previous value.
+        /// </param>
+        /// <typeparam name="TIn">The input type.</typeparam>
+        /// <typeparam name="TOut">The output type.</typeparam>
+        /// <typeparam name="TMat">The materialization type.</typeparam>
+        /// <remarks>
+        /// This is designed to allow fan-in stages where output from one of the sources is intermittent / infrequent
+        /// and users just want the previous value to be reused.
+        /// </remarks>
+        public static Flow<TIn, TOut, TMat> RepeatPrevious<TIn, TOut, TMat>(this Flow<TIn, TOut, TMat> flow, Action<TOut, TOut> onItemChanged)
+        {
+            return flow.Via(new ReuseLatest<TOut>(onItemChanged));
```

Comment:
Why not named as `RepeatLatest` just as the stage?

Reply:
I got feedback on name changes and didn't propagate it everywhere I guess, although the stage implementation is really meant to be internal.